kubernetes/start

```diff
@@ -71,5 +76,23 @@ fi
 sudo mkdir -p $PWD/tmp/mariadb-socket
 sudo chmod 777 $PWD/tmp/mariadb-socket
 
+if docker images | grep "^sles12/velum" &> /dev/null ; then
+  log "Velum development image is present"
+else
+  log "Building Velum development image..."
+  CONTAINER_BUILD=`docker run -v $(readlink -f $PWD/..):/srv/velum -d docker-testing-registry.suse.de/sles12/velum:development \
```

Comment:
No, because this image contains the dependencies required for a `bundle install` in development to succeed. Note that in development we require more gems than in production, so there are some packages that we need in order to successfully run `bundle install` from scratch: https://build.suse.de/package/view_file/home:ereslibre:branches:Devel:CASP:1.0:ControllerNode/sles12sp2-velum-docker-image/sles12sp2-velum-docker-image.kiwi?expand=1